build.gradle.kts

```diff
@@ -230,7 +230,7 @@ tasks.withType<BootBuildImage> {
     environment = mapOf("BP_HEALTH_CHECKER_ENABLED" to "true")
     buildpacks = listOf(
         "urn:cnb:builder:paketo-buildpacks/java",
-        "gcr.io/paketo-buildpacks/health-checker",
+        "gcr.io/paketo-buildpacks/health-checker"
```

Comment:
I think the trailing `,` was done deliberately so when additional lines are added the PR diff is slightly simpler.

Reply:
For some reason the build failed for ktlint 11.3.0, so had to remove them . Let me put that back on for ktlint `11.0.0`